tslint.json

```diff
@@ -73,7 +73,6 @@
     "no-trailing-whitespace": true,
     "no-unnecessary-initializer": true,
     "no-unused-expression": true,
-    "no-use-before-declare": true,
```

Comment:
we do you remove this rule, it's better to declare before use

Reply:
According to the pull request [#4695](https://github.com/palantir/tslint/pull/4695) 'no-use-before-declare' had been deprecated, we don't need this rule for typescript 2.9+.